aten/src/ATen/native/Math.h

```diff
@@ -67,6 +68,84 @@ Output was modified to be inf or -inf when input is 1 or -1. */
     POSSIBILITY OF SUCH DAMAGE.
 */
 
+namespace {
+/*
+ * This function is derived from the implementation of the i0e function in the
+ * Cephes Math Library. See note [3-Clause BSD License for the Cephes Math
+ * Library].
+ *
+ * Computes an approximation of the exponentially scaled zeroth order modified
+ * Bessel function of the first kind. The approximation is actually two
+ * (sub)approximations, both using a Chebyshev polynomial expansion. One
+ * approximates the function over [0, 8], and the other over (8, infinity). This
+ * function takes the absolute value of all inputs to convert them into the
+ * domain of the approximation.
+ */
+jiterator_code_stringify(
```

Comment:
In the future I would write `jiterator_code(` on the same line as `jiterator_code_stringify` to avoid the extra level of indentation